src/Jira/Api.php

```diff
@@ -68,27 +68,34 @@ class Api
 	protected $options = self::AUTOMAP_FIELDS;
 
 	/**
-	 * Fields.
+	 * Client-side cache of Fields.
 	 *
-	 * @var array
+	 * @var array|null List of fields when loaded, null when nothing is fetched yet
```

Comment:
In multi-line notation of `@var` DocBlock the description should go in dedicated place (up). In single line `@var` DocBlock notation indeed the description would be written right after type name.